internal/stocker/aws.go

```diff
@@ -75,7 +81,7 @@ func (s AWSStocker) getRegions() []string {
 
 	regions, err := ec2Client.DescribeRegions(&ec2.DescribeRegionsInput{})
 	if err != nil {
-		log.Fatal(err.Error())
+		logger.Warn("Failed to retrieve AWS regions", zap.String("reason", err.Error()))
```

Comment:
If we can't retrieve AWS regions, we cannot scan them for looking up instances. I think this should have a higher level. At least Error, or Fatal instead

Reply:
Ok, sounds good